sci-biology/bowtie/bowtie-1.1.2.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils toolchain-funcs
+
+DESCRIPTION="Popular short read aligner for Next-generation sequencing data"
+HOMEPAGE="http://bowtie-bio.sourceforge.net/"
+SRC_URI="mirror://sourceforge/bowtie-bio/${P}-src.zip"
+
+LICENSE="Artistic"
+SLOT="1"
+KEYWORDS="~amd64 ~x86 ~x64-macos"
+
+IUSE="examples tbb"
+
+DEPEND="app-arch/unzip
+	tbb? ( dev-cpp/tbb )"
+RDEPEND=""
+
+DOCS=( AUTHORS NEWS TUTORIAL doc/README )
+HTML_DOCS=( doc/{manual.html,style.css} )
+
+src_prepare() {
+	# Suppress useless -Wall pollution
+	sed -i 's/\-Wall/\-Wno-enum-compare/g' Makefile || die
```

Comment:
It's not for the maintainer to make the build log pretty by removing warning flags, really. Just leave them untouched, until they actually cause trouble.